### ISSUE TITLE
Add an right alt/option key

### ISSUE THREAD

**Issue by MidnightRocket** [BlueM/cliclick#88]
I use cliclick to type text into Parallels with a Linux VM running, which works great. But when I try to get cliclick to type special characters like `£ ∞ ™`, I get nothing in my VM, because cliclick uses the left alt/option key as modifier key, and then Linux or Parallels thinks I am trying to use a command. It work when I manually types in these character using right alt/option key. 

Maybe you could change the alt/option modifier key to the right alt/option key, as the default, because I believe this would work across all of MacOS, including VM's. 

I don't hope this is too difficult to implement/change.

**Comment by BlueM** [comment]
When you add a line …

```objectivec
@"61", @"ralt",
```

… below https://github.com/BlueM/cliclick/blob/master/Actions/KeyDownUpBaseAction.m#L39 and compile it, does it work with Parallels when using `ralt` instead of `alt`?

**Comment by MidnightRocket** [comment]
Thank you for the response. 
So I have looked into it, and it works. 
I have changed it around, so that `lalt` is left alt-key, and `alt` is right alt, and therefore the default alt-key. 
I then only need to type in this 
`cliclick t:™£∞`
instead of specify the modifier key manually. 

Here are the changes https://github.com/MidnightRocket/cliclick/blob/9d3789fb9f372a84eee31fec5dd1e055d82650f4/Actions/KeyDownUpBaseAction.m#L39-L40
